local-containers/docker-compose.override.yml

```diff
@@ -32,16 +31,41 @@ services:
       - nodejs
     labels:
       - "traefik.enable=true"
-      - "traefik.http.routers.rendering-secure.entrypoints=websecure"
-      - "traefik.http.routers.rendering-secure.rule=Host(`${RENDERING_HOST}`)"
-      - "traefik.http.routers.rendering-secure.tls=true"
+      - "traefik.http.routers.rendering-secure-nextjs.entrypoints=websecure"
+      - "traefik.http.routers.rendering-secure-nextjs.rule=Host(`${RENDERING_HOST_NEXTJS}`)"
+      - "traefik.http.routers.rendering-secure-nextjs.tls=true"
+
+  angular:
```

Comment:
I would name both angular and nextjs services like `rendering-angular` and `rendering-nextjs` or smth similar to make it more clear. Plus, it will make more clear internal Docker urls, e.g. for now they are "http://nextjs:3000"  and "http://angular:3000"

Reply:
makes sense